test_gr4vy.py

```diff
@@ -25,20 +25,17 @@
 )
 from gr4vy.gr4vy_api.openapi_client.model.transaction_request import TransactionRequest
 
-gr4vy_id = "spider"
+gr4vy_id = "example009"
 private_key_location = "./private_key.pem"
-
-client = Gr4vyClient(gr4vy_id, private_key_location)
-client.client.configuration.debug = True
-global buyer_id, payment_method_id, payment_service_id, transaction_id
-
+environment = "sandbox"
+client = Gr4vyClient(gr4vy_id, private_key_location, environment=environment)
 
 def testCreateClient():
     assert client
 
 
 def testGr4vyClientWithBaseUrl():
-    assert Gr4vyClientWithBaseUrl("https://spider.gr4vy.app", "private_key.pem")
+    assert Gr4vyClientWithBaseUrl("https://{}.gr4vy.app".format(gr4vy_id), "private_key.pem", environment)
```

Comment:
What does this test? Just that it can be created?

Reply:
Yes this just tests the overall method which at this point just creates the client.

Reply:
Ok. Might be good to also test that the BaseURL is set right? I assume we do the same with the regular constructor?